redwood-flex-container/src/commonMain/kotlin/app/cash/redwood/flexcontainer/properties.kt

```diff
@@ -185,29 +184,38 @@ public value class JustifyContent private constructor(private val ordinal: Int)
 /**
  * A MeasureSpec encapsulates the layout requirements passed from parent to child.
  * Each MeasureSpec represents a requirement for either the width or the height.
- * A MeasureSpec is comprised of a size and a mode.
+ * A MeasureSpec is composed of a size and a mode.
  */
-@JvmInline
-public value class MeasureSpec private constructor(private val value: Int) {
+public class MeasureSpec private constructor(
```

Comment:
We can't occupy the top 2 bits with a `Double` unfortunately (at least easily?). Will circle back on seeing if this can be an inline class later.

Reply:
You'll have to interpret the bits as a Long, mask the bits, and then interpret back as a double. File a tracking issue?